### PR TITLE
Increase sprite size by 50%

### PR DESCRIPTION
**Pull request by nightblade9**
Recalibrate sight/vision appropriately
Extract monster vision into const

### CODE REVIEW

**Review by 3akev — APPROVED** [review]
LGTM

File: src/project.godot [hunk 1]
```diff
@@ -21,6 +21,7 @@ config/icon="res://icon.png"
 
 [autoload]
 
+Features="*res://Scripts/Features.gd"
```
Comment (3akev) [inline src/project.godot]: Was this intentional or is it godot acting up again?

Comment (nightblade9) [reply src/project.godot]: Godot acting up again.

Comment (nightblade9) [reply src/project.godot]: Fixed.

**Pull request merged**